Formula/gitslave.rb

```diff
@@ -16,6 +16,8 @@ class Gitslave < Formula
     sha256 cellar: :any_skip_relocation, all:           "f0dc05bf4e295a2a092e5e2df35d108fa33e1b9ebf89c17965abc69ac2897851"
```

Comment:
Side note: `all` bottle should be manually removed and new Monterey bottle job needs to be triggered.
```suggestion
```